tutorials/2d/particle_systems_2d.rst

```diff
@@ -204,6 +204,11 @@ parameter must be greater than 0.
 
 .. image:: img/paranim3.gif
 
+Flatness
+~~~~~~~~
+
+This property is only useful for 3D particles.
```

Comment:
It might be useful to explain what does this property does when you increase/decrease its value, as well as documenting the allowed range (which IIRC is 0..1).

Reply:
That's the thing though. The property only does something in 3D scenes and this page is for 2D particles only. To me it doesn't make sense to go over it here. And it is documented in the class reference.